code/tools/r68k/firmware/trap14.asm

```diff
@@ -52,6 +52,7 @@ TRAP_14_HANDLER::
     dc.l    .EPILOGUE                   ; FC == 14 ; RESERVED
     dc.l    .EPILOGUE                   ; FC == 15 ; RESERVED
     dc.l    .DEV_CTRL                   ; FC == 16 ; DEV_CTRL if so...
+    dc.l    .RECVCHAR                   ; FC == 17 ; RECVCHAR if so...
```

Comment:
Could we change the comment to indicate that this is the `INPUTCHAR` trap, just to save confusion in the future? Also, should we add function `18` (`CHECKINPUT`) while we're here? It should just be another one-line change (it's the same as the `CHECKCHAR` call on r68k).